### PR TITLE
Final pull request of the project

### PR DESCRIPTION
**Pull request by trillianjose**
Waiting for review

### CODE REVIEW

File: app/helpers/sessions_helper.rb [hunk 1]
```diff
@@ -0,0 +1,57 @@
+module SessionsHelper
+
+  def log_in(user)
```
Comment (zlayabekrija) [inline app/helpers/sessions_helper.rb]: `log_in` and `log_in_as` are absolutely the same methods, please keep only one of those two.

File: config/routes.rb [hunk 2]
```diff
@@ -1,3 +1,15 @@
...
+
+  get       '/signup',  to: 'users#new'
+  post      '/signup',  to: 'users#create'
+  get       '/login',   to: 'sessions#new'
+  post      '/login',   to: 'sessions#create'
+  delete    '/logout',  to: 'sessions#destroy'
+  resources 'users',    only: [:new, :create, :show]
+  resources 'sessions', only: [:new]
+  resources 'events',   only: [:index, :new, :create] do
+    post :checkin
```
Comment (zlayabekrija) [inline config/routes.rb]: since both routes belong to Attendance model, they should be built within attendance controller.

File: db/seeds.rb [hunk 3]
```diff
@@ -5,3 +5,16 @@
 #
 #   movies = Movie.create([{ name: 'Star Wars' }, { name: 'Lord of the Rings' }])
 #   Character.create(name: 'Luke', movie: movies.first)
+
```
Comment (zlayabekrija) [inline db/seeds.rb]: To make your life easier with seeding, try to use Faker gem. ***This is only suggestion, it's not mandatory***

File: app/controllers/events_controller.rb [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+      @event.create_owner_relationship(user_id: current_user.id, role: 'owner')
+
+      flash[:info] = 'Successfully created an event'
+      redirect_to :root
+    else
+      render 'new'
+    end
+  end
+
+  def checkin
```
Comment (zlayabekrija) [inline app/controllers/events_controller.rb]: `checkin` and `assistents` methods as said are part of Attendance model (both of them are making records in the Attendance table with event.attneddace build option). Please move both of the methods to the appropriate controller.

**Review by zlayabekrija — CHANGES_REQUESTED** [review]
Working good, meeting requirements, really good job.
There are some minor changes, please follow up with comments and make those changes. If any help or clarification is needed feel free to contact me via slack.
Resubmit review request when ready.
Happy codding.

File: app/helpers/sessions_helper.rb (hunk 1 above)
Comment (shshamim63) [reply app/helpers/sessions_helper.rb]: deleted repeating method

File: config/routes.rb (hunk 2 above)
Comment (shshamim63) [reply config/routes.rb]: moved the  methods.

File: app/controllers/events_controller.rb (hunk 4 above)
Comment (shshamim63) [reply app/controllers/events_controller.rb]: moved those methods.

**Comment by dipto0321** [comment]
Hi @shshamim63 and @trillianjose ,

So far so good. This is from me 👍 . You can merge it into the master.

Thanks 🙏

**Pull request merged**